pkg/handler/processor/ite6/ite6_test.go

```diff
@@ -0,0 +1,254 @@
+				"builder": { "id": "https://github.com/Attestations/GitHubHostedActions@v1" },
+				"buildType": "https://github.com/Attestations/GitHubActionsWorkflow@v1",
+				"invocation": {
+				  "configSource": {
+					"uri": "git+https://github.com/curl/curl-docker@master",
+					"digest": { "sha1": "d6525c840a62b398424a78d792f457477135d0cf" },   
+					"entryPoint": "build.yaml:maketgz"
+				  }
+				},
+				"metadata": {
+				  "completeness": {
+					  "environment": true
+				  }
+				},
+				"materials": [
+				  {
+					"uri": "git+https://github.com/curl/curl-docker@master",
+					"digest": { "sha1": "d6525c840a62b398424a78d792f457477135d0cf" }
+				  }, {
+					"uri": "github_hosted_vm:ubuntu-18.04:20210123.1"
+				  }
+				]
+			}
+	}`
+
+	predicate = v02.ProvenancePredicate{
+		Builder: v02.ProvenanceBuilder{
+			ID: "https://github.com/Attestations/GitHubHostedActions@v1",
+		},
+		BuildType: "https://github.com/Attestations/GitHubActionsWorkflow@v1",
+		Invocation: v02.ProvenanceInvocation{
+			ConfigSource: v02.ConfigSource{
+				URI: "git+https://github.com/curl/curl-docker@master",
+				Digest: v02.DigestSet{
+					"sha1": "d6525c840a62b398424a78d792f457477135d0cf",
+				},
+				EntryPoint: "build.yaml:maketgz",
+			},
+		},
+		Metadata: &v02.ProvenanceMetadata{
+			Completeness: v02.ProvenanceComplete{
+				Environment: true,
+			},
+		},
+		Materials: []v02.ProvenanceMaterial{
+			{
+				URI: "git+https://github.com/curl/curl-docker@master",
+				Digest: v02.DigestSet{
+					"sha1": "d6525c840a62b398424a78d792f457477135d0cf",
+				},
+			},
+			{
+				URI: "github_hosted_vm:ubuntu-18.04:20210123.1",
+			},
+		},
+	}
+)
+
+func TestITE6Processor_ValidateSchema(t *testing.T) {
+	tests := []struct {
+		name    string
+		args    *processor.Document
+		wantErr bool
+	}{
+		{
+			name: "ITE6 Doc with unknown payload",
+			args: &processor.Document{
+				Blob:   []byte(badProvenance),
+				Type:   processor.DocumentITE6,
+				Format: processor.FormatJSON,
+				SourceInformation: processor.SourceInformation{
+					Collector: "TestCollector",
+					Source:    "TestSource",
+				},
+			},
+			wantErr: true,
+		},
+		{
+			name: "ITE6 Doc with valid payload",
+			args: &processor.Document{
+				Blob:   []byte(ite6SLSA),
+				Type:   processor.DocumentITE6,
+				Format: processor.FormatJSON,
+				SourceInformation: processor.SourceInformation{
+					Collector: "TestCollector",
+					Source:    "TestSource",
+				},
+			},
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			e := &ITE6Processor{}
+			if err := e.ValidateSchema(tt.args); (err != nil) != tt.wantErr {
+				t.Errorf("ITE6Processor.ValidateSchema() error = %v, wantErr %v", err, tt.wantErr)
+			}
+		})
+	}
+}
+
+func TestITE6Processor_Unpack(t *testing.T) {
+	predicatePayload, _ := json.Marshal(predicate)
+	tests := []struct {
+		name    string
+		args    *processor.Document
+		want    []*processor.Document
+		wantErr bool
+	}{
+		{
```

Comment:
minor nit: you can bring this bracer up to the previous line so that it will move back 1 tab to make it easier to read

Reply:
done.